src/test/resources/input_CNMResponse.json

```diff
@@ -0,0 +1,70 @@
+{
+  "input": {
```

Comment:
The input here isn't a valid CNM response. It doesn't contain response or processCompleteTime.